Microsoft.Toolkit.Forms.UI.Controls.WebView/WebBaseCompatibilityAdapter.cs

```diff
@@ -13,8 +13,10 @@ namespace Microsoft.Toolkit.Forms.UI.Controls
     internal abstract class WebBaseCompatibilityAdapter : DependencyObject, IWebViewCompatibleAdapter
     {
         protected WebBaseCompatibilityAdapter()
-        {
-            Initialize();
+        {
+#pragma warning disable CA2214 // Do not call overridable methods in constructors

```

Comment:
Is there a bug open for this? @GraniteStateHacker received feedback to fix on initial PR but it never made it through

Reply:
I don't think we have a bug opened for it, but this class is internal. Is that a real issue?